src/services/helpers.ts

```diff
@@ -129,3 +129,5 @@ export const loadTrainingsSet = async (disciplineId: number): Promise<ServerResp
   const trainingSet = await getFromEndpoint<ServerResponseDiscipline>(trainingSetUrl)
   return trainingSet
 }
+
+export const getLabels = (): any => labels
```

Comment:
Doesn't this remove type annotations? Intellij currently tells me whether a label exists or does not.

Reply:
You are right, added appropriate type.